microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,96 @@
+---
+title: OS Regional Data Display in Microsoft Edge
+description: How users and web developers can use the OS regional format in Microsoft Edge for improved site experiences
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/15/2022
+--- 
+# Introducing the OS regional sharing setting
```

Comment:
@JasonAndrewWriter - make sure all copies of article title are updated:
toc.yml
title: yaml field
h1
link text of articles linking here, probably none

Reply:
it looks like it was done.